lib/daemon_runner/retry_errors.rb

```diff
@@ -0,0 +1,22 @@
+require 'retryable'
+
+module DaemonRunner
+    #
+    # Retry Errors
+    #
+    class RetryErrors
+      extend Logger
+
+      class << self
+        def retry(retries: 3, exceptions: [Faraday::ClientError], &block)
+          Retryable.retryable({
```

Comment:
STYLE: The `{}` should be unnecessary here. You should be able define the properties on one line with the opening `do` of the code block and make the block a lot more readable.